src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -2,77 +2,92 @@
 // Licensed under the MIT license.
 
 using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Utils;
 
 namespace Microsoft.PowerFx
 {
-    [Flags]
-#pragma warning disable CA2217 // Do not mark enums with FlagsAttribute
-    public enum Features : int
-#pragma warning restore CA2217 // Do not mark enums with FlagsAttribute
+    public sealed class Features
     {
-        None = 0x0,
-
         /// <summary>
         /// Enable Table syntax to not add "Value:" extra layer.
-        /// Added on 1st July 2022.
         /// </summary>
-        TableSyntaxDoesntWrapRecords = 0x1,
+        internal bool TableSyntaxDoesntWrapRecords { get; set; }
```

Comment:
>set; [](http://example.com/codeflow?start=58&length=4)

Setters can be private here. That way the class can control the combinations of these being set correctly.

Reply:
Either we have internal getters / setters, or we need to have a constructor (internal) that Power Apps can use to selectively set those values. Since those are mostly temporary, I don't see a reason to make it more complex.